server/autolink/lib_jira_test.go

```diff
@@ -80,8 +80,8 @@ var jiraTests = []linkTest{
 	}, {
 		"Not relinking",
 		autolink.Autolink{
-			Pattern:  "(?P<project_id>\\w+)(-)(?P<jira_id>\\d+)",
-			Template: "[${project_id}-${jira_id}](https://mattermost.atlassian.net/browse/${project_id}-${jira_id})",
+			Pattern:  "(MM)(-)(?P<jira_id>\\d+)",
+			Template: "[MM-$jira_id](https://mattermost.atlassian.net/browse/MM-$jira_id)",
```

Comment:
It seems we should still wrap this in curly braces, even if not necessary, to keep the same convention as usual

```suggestion
			Template: "[MM-$jira_id](https://mattermost.atlassian.net/browse/MM-${jira_id})",
```